packages/types/src/codec/BTreeMap.ts

```diff
@@ -0,0 +1,183 @@
+import { Compact, U8a } from '.';
+import { compareMap, decodeU8a } from './utils';
+import { AnyJson, Constructor, Codec, IHash } from '../types';
```

Comment:
Put the types import first, then the rest.

Additionally - first the package imports, then our won.